src/sessions.rs

```diff
@@ -47,7 +47,17 @@ fn assert_socket(name: &str) -> bool {
     }
 }
 
-fn print_sessions(sessions: Vec<String>) {
+pub(crate) fn get_sessions() -> Vec<String> {
```

Comment:
This function looks redundant to me now. We can simply `unwrap()` on the result returned by `read_sessions()` to get the same behaviour.